src/pretix/static/pretixbase/scss/_variables.scss

```diff
@@ -40,14 +40,18 @@ $navbar-inverse-link-hover-color: $gray-lighter;
 $navbar-inverse-brand-hover-color: $gray-lighter;
 $navbar-inverse-color: white;
 
-$state-success-bg:               white !default;
-$state-success-border:           $brand-success;
-$state-info-bg:                  white !default;
-$state-info-border:              $brand-info;
-$state-warning-bg:               white !default;
-$state-warning-border:           $brand-warning;
-$state-danger-bg:                white !default;
-$state-danger-border:            $brand-danger;
+$state-success-bg:               lighten($brand-success, 48%) !default;
+$state-success-border:           lighten($brand-success, 30%);
+$state-success-text:             darken($brand-success, 15%);
+$state-info-bg:                  lighten($brand-info, 33%) !default;
+$state-info-border:              lighten($brand-info, 20%);
+$state-info-text:                darken($brand-info, 20%);
+$state-warning-bg:               lighten($brand-warning, 41%) !default;
+$state-warning-border:           lighten($brand-warning, 20%);
+$state-warning-text:             darken($brand-warning, 25%);
+$state-danger-bg:               lighten($brand-danger, 43%) !default;
```

Comment:
Very weirdly, this works for me in control, but not in presale:

![20210929_171653](https://user-images.githubusercontent.com/64280/135298429-35e3ccc9-e809-48b8-9338-18a3a1706070.png)